format.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="format.css">
+    <title>National Geo</title>
+</head>
+<body>
+    <div class="container">
+        <h2>Days, Months and Seasons</h2>
+        <!-- unordered list - days of the week -->
+        <ul>
+            <li class="days">Sunday</li>
+            <li class="days">Monday</li>
+            <li class="days">Tuesday</li>
+            <li class="days">Wednesday</li>
+            <li class="days">Thursday</li>
+            <li class="days">Friday</li>
+            <li class="days">Saturday</li>
+        </ul>
+        <hr>
+        <!-- ordered lists - months of the year -->
+        <ol>
+            <li class="months">January</li>
+            <li class="months">February</li>
+            <li class="months">March</li>
+            <li class="months">April</li>
+            <li class="months">May</li>
+            <li class="months">June</li>
+            <li class="months">July</li>
+            <li class="months">August</li>
+            <li class="months">September</li>
+            <li class="months">October</li>
+            <li class="months">November</li>
+            <li class="months">December</li>
+        </ol>
+        <hr>
+        <!-- definition list - seasons -->
+        <dl>
+            <li class="seasons">Spring</li>
+            <li class="seasons">Summer</li>
+            <li class="seasons">Fall</li>
+            <li class="seasons">Winter</li>
+        </dl>
+    </div>
+    <div class="container2">
+        <h2>Seasons and their months</h2>
+        <!-- unordered lists - seasons again -->
+        <ul>
+            <li>Spring</li>
+            <!-- ordered lists - months of Spring -->
+                <ol>
+                    <li>March</li>
+                    <li>April</li>
+                    <li>May</li>
+                </ol>
+            <li>Summer</li>
+            <!-- ordered lists - months of Summer -->
+                <ol>
+                    <li>June</li>
+                    <li>July</li>
+                    <li>August</li>
+                </ol>
+            <li>Fall</li>
+            <!-- ordered lists - months of Fall -->
+                <ol>
+                    <li>September</li>
+                    <li>October</li>
+                    <li>November</li>
+                </ol>
+            <li>Winter</li>
+            <!-- ordered lists - months of Winter -->
+                <ol>
+                    <li>December</li>
+                    <li>January</li>
+                    <li>February</li>
+                </ol>
+        </ul>
```

Comment:
all good

Reply:
yes, UI looks good. it would be great if u can increase the font size a little bit. overall good.